### ISSUE TITLE
Performance - Prevent toString evaluation in Velocity

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#11441]
The tostring check is an unnecessary double check and it looks like this in a threaddump - which is obviously an expensive operation.  

```

"url:GET//demo.dotcms.com/about-us/index | lang:1 | ip:0:0:0:0:0:0:0:1 | Admin:false | start:04-28-2017 04:44:21 EDT {live/36b3f2a1-35c7-4ae0-9f24-526063662ed6_694805c6-8f07-467f-a434-0da04a93e9e5.field}" daemon prio=5 tid=0x00007fdc84fe5000
  java.lang.Thread.State: RUNNABLE
	at java.util.Objects.hashCode(Objects.java:98)
	at java.util.HashMap$Node.hashCode(HashMap.java:296)
	at java.util.AbstractMap.hashCode(AbstractMap.java:530)
	at java.util.AbstractList.hashCode(AbstractList.java:541)
	at java.util.concurrent.ConcurrentHashMap.hashCode(ConcurrentHashMap.java:1292)
	at java.util.HashMap.hash(HashMap.java:338)
	at java.util.HashMap.put(HashMap.java:611)
	at java.util.HashSet.add(HashSet.java:219)
	at com.dotcms.repackage.org.apache.commons.lang.builder.ToStringStyle.register(ToStringStyle.java:182)
	at com.dotcms.repackage.org.apache.commons.lang.builder.ToStringStyle.appendInternal(ToStringStyle.java:460)
	at com.dotcms.repackage.org.apache.commons.lang.builder.ToStringStyle.append(ToStringStyle.java:428)
	at com.dotcms.repackage.org.apache.commons.lang.builder.ToStringBuilder.append(ToStringBuilder.java:840)
	at com.dotcms.repackage.org.apache.commons.lang.builder.ReflectionToStringBuilder.appendFieldsIn(ReflectionToStringBuilder.java:606)
	at com.dotcms.repackage.org.apache.commons.lang.builder.ReflectionToStringBuilder.toString(ReflectionToStringBuilder.java:762)
	at com.dotcms.repackage.org.apache.commons.lang.builder.ReflectionToStringBuilder.toString(ReflectionToStringBuilder.java:287)
	at com.dotcms.repackage.org.apache.commons.lang.builder.ReflectionToStringBuilder.toString(ReflectionToStringBuilder.java:121)
	at com.dotcms.repackage.org.apache.commons.lang.builder.ToStringBuilder.reflectionToString(ToStringBuilder.java:126)
	at com.dotmarketing.portlets.contentlet.model.Contentlet.toString(Contentlet.java:238)
	at java.lang.String.valueOf(String.java:2994)
	at java.lang.StringBuffer.append(StringBuffer.java:265)
	- locked <0x000000005f288daf> (a java.lang.StringBuffer)
	at com.dotcms.repackage.org.apache.commons.lang.builder.ToStringStyle.appendDetail(ToStringStyle.java:578)
	at com.dotcms.repackage.org.apache.commons.lang.builder.ToStringStyle.appendInternal(ToStringStyle.java:542)
	at com.dotcms.repackage.org.apache.commons.lang.builder.ToStringStyle.append(ToStringStyle.java:428)
	at com.dotcms.repackage.org.apache.commons.lang.builder.ToStringBuilder.append(ToStringBuilder.java:840)
	at com.dotcms.repackage.org.apache.commons.lang.builder.ReflectionToStringBuilder.appendFieldsIn(ReflectionToStringBuilder.java:606)
	at com.dotcms.repackage.org.apache.commons.lang.builder.ReflectionToStringBuilder.toString(ReflectionToStringBuilder.java:759)
	at com.dotcms.repackage.org.apache.commons.lang.builder.ReflectionToStringBuilder.toString(ReflectionToStringBuilder.java:287)
	at com.dotcms.repackage.org.apache.commons.lang.builder.ReflectionToStringBuilder.toString(ReflectionToStringBuilder.java:121)
	at com.dotcms.repackage.org.apache.commons.lang.builder.ToStringBuilder.reflectionToString(ToStringBuilder.java:126)
	at com.dotmarketing.viewtools.content.ContentMap.toString(ContentMap.java:431)
	at org.apache.velocity.runtime.parser.node.ASTReference.evaluate(ASTReference.java:552)
	at org.apache.velocity.runtime.parser.node.ASTAndNode.evaluate(ASTAndNode.java:104)
	at org.apache.velocity.runtime.parser.node.ASTExpression.evaluate(ASTExpression.java:62)
	at org.apache.velocity.runtime.parser.node.ASTIfStatement.render(ASTIfStatement.java:85)
	at org.apache.velocity.runtime.parser.node.SimpleNode.render(SimpleNode.java:339)
	at org.apache.velocity.runtime.directive.Parse.render(Parse.java:264)
	at org.apache.velocity.runtime.parser.node.ASTDirective.render(ASTDirective.java:211)
	at org.apache.velocity.runtime.parser.node.ASTBlock.render(ASTBlock.java:72)
```

**Comment by jgambarios** [comment]
PR: https://github.com/dotCMS/core/pull/11452

**Comment by bryanboza** [comment]
Fixed, tested on release-4.1 // MySQL // FF
